docs/docs/06-migration/v28.0.0.md

```diff
@@ -0,0 +1,124 @@
+---
+sidebar_position: 990
+title: v28.0.0
+description: For chains that were scaffolded with Ignite CLI versions lower than v28.0.0 changes are required to use Ignite CLI v28.0.0
+---
+
+## **Upgrade to v28.0.0 - New Versioning Scheme in Ignite**
+
+With the latest update, Ignite has transitioned its versioning format from a leading-zero release system to a full number release system. This change marks a significant shift in how we communicate updates and stability in our software. Where the previous version was denoted as v0.27.0, it will now be upgraded to v28.0.0.
+
+This new versioning approach enhances our version control by clearly indicating major, minor, and patch releases. 
+From now on first number indicates a major release with breaking API changes, second number indicates minor release that might include new features while the last number is typically focused on bug fixes and minor improvements.
+[Learn more about semantic versioning](https://semver.org/).
+
+## **Plugins are now called Apps. Upgrade Configuration Files**
+
+Ignite `v28.0.0` changes the plugin system which is now called Ignite Apps. This version includes changes
+to the CLI command names and the plugin configuration file.
+
+The plugins configuration file is now called `igniteapps.yml` and "plugins" are now called "apps".
+
+The plugins configuration home directory is now `$HOME/.ignite/apps` instead `$HOME/.ignite/plugins`.
+
+Updates can be automatically applied by running `ignite doctor` in your blockchain application directory.
+Running the command outside your blockchain application directory will only update the global plugins.
+
+## **Ignite and Cosmos SDK Upgrade Guide: From Ignite v0.27.0 to v28.0.0 and Cosmos SDK v0.47 to v0.50**
+
+### **Introduction**
+
+This guide provides a step-by-step process for developers to upgrade their applications from Ignite version 0.27.0 to 28.0.0, along with an upgrade in the Cosmos SDK from version 0.47 to v0.50. It covers essential changes, new features, and adjustments required for a smooth transition.
+
+### **Prerequisites**
+
+- Backup your current project.
+- Ensure you have Ignite v0.27.0 and Cosmos SDK v0.47 installed.
+- Basic familiarity with command line operations and the existing project structure.
+
+### **Step 1: Update Ignite CLI to Version 28.0.0**
+
+- **Command**: Run **`curl https://get.ignite.com/cli@v28.0.0 | bash`** in your terminal.
+- **Note**: This command updates the Ignite CLI to the latest version. Ensure you have the necessary permissions to execute it.
+
+### **Step 2: Update Scaffold Chain Command**
+
+- **Old Command**: **`ignite scaffold chain github.com/alice/blog`**
+- **New Command**: **`ignite scaffold chain blog`**
```

Comment:
Mmh, that does not look correct. You can use both. One will create you a module name with a GitHub address and the other one without.
Imho, we should recommended the other one.

Reply:
Interesting. So now both is possible but the one prepend with `github.com/user/` is recommended?